index.html

```diff
@@ -11,58 +11,59 @@
-            <option>Saracens</option>
-            <option>Slavs</option>
-            <option>Spanish</option>
-            <option>Tatars</option>
-            <option>Teutons</option>
-            <option>Turks</option>
-            <option>Vietnamese</option>
-            <option>Vikings</option>
-        </select>
-        <div id="info">Version: Age of Empires II DE Update <a href="https://www.ageofempires.com/news/aoe2de-update-35584/" target="_blank" title="Changelog">35584</a>  |
-            <a href="https://github.com/SiegeEngineers/aoe2techtree" target="_blank">GitHub</a>
-        </div>
-    </div>
     <div id="wrapper">
         <div id="metainfo">
             <div id="civinfo">
                 <div id="civtitle">
-                    <h2 id="civname">Loading…</h2>
+                    <select id="civselect" onchange="loadCiv()">
+                        <option>Aztecs</option>
+                        <option>Berbers</option>
+                        <option>Britons</option>
+                        <option>Bulgarians</option>
+                        <option>Burmese</option>
+                        <option>Byzantines</option>
+                        <option>Celts</option>
+                        <option>Chinese</option>
+                        <option>Cumans</option>
+                        <option>Ethiopians</option>
+                        <option>Franks</option>
+                        <option>Goths</option>
+                        <option>Huns</option>
+                        <option>Incas</option>
+                        <option>Indians</option>
+                        <option>Italians</option>
+                        <option>Japanese</option>
+                        <option>Khmer</option>
+                        <option>Koreans</option>
+                        <option>Lithuanians</option>
+                        <option>Magyars</option>
+                        <option>Malay</option>
+                        <option>Malians</option>
+                        <option>Mayans</option>
+                        <option>Mongols</option>
+                        <option>Persians</option>
+                        <option>Portuguese</option>
+                        <option>Saracens</option>
+                        <option>Slavs</option>
+                        <option>Spanish</option>
+                        <option>Tatars</option>
+                        <option>Teutons</option>
+                        <option>Turks</option>
+                        <option>Vietnamese</option>
+                        <option>Vikings</option>
+                    </select>
                     <img id="civlogo" height=52 width=52 data-transparent="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII=">
                 </div>
                 <hr>
                 <p id="civtext"></p>
             </div>
+            <div id="version">
```

Comment:
Can we reorganise the `version` and `thanks` part? I would like to have three blocks there, all with equal font size and a bit of space in between (approximately one line?):

```
[version]

[github]

[thanks]
[thanks]
[thanks]
```

To me, that sounds like a sequence of three full-width divs.

We can remove the github link from the thanks section (on "Made by").

We can also remove the commented-out descriptions link now.

Reply:
Something like this? Adjusted some text to make each line fit in 1 line at 8pt font.
![image](https://user-images.githubusercontent.com/6392995/76153168-0a682f80-607d-11ea-92f7-eba4b98af997.png)